ocp_resources/node_network_configuration_policy.py

```diff
@@ -322,10 +323,41 @@ def _absent_interface(self):
         if self.ports:
             self.add_ports()
 
+        initial_success_status_time = self._get_last_successful_transition_time()
         ResourceEditor(
             patches={self: {"spec": {"desiredState": {"interfaces": self.desired_state["interfaces"]}}}}
         ).update()
 
+        # If the NNCP failed on setup, then at this point it's not in AVAILABLE status, so the next time it will be in
+        # AVAILABLE status will necessarily be the updated one.
+        if initial_success_status_time:
```

Comment:
```suggestion
        if initial_success_status_time := self._get_last_successful_transition_time()
```
and remove line 326

Reply:
I can't.
If I do that, it will miss the purpose of this PR :)
Storing initial_transition_time must be done before calling `update()`.

Reply:
Please add a comment above line 326 about this